sm-cw-lambda-sns-terraform/example-pattern.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title": "AWS Secrets Manager to CloudWatch Events to Lambda to SNS",
+  "description": "An terraform template that detects and notifies on Amazon Secrets Manager secret key creation, Updation and deletion using Amazon CloudWatch event and Amazon SNS",
+  "language": "YAML",
+  "level": "200",
+  "framework": "Terraform",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "Once the template deployed successfully, first thing to do is to confirm the Email subscription. You will receive an email to confirm it.  Then, head to AWS Secrets Manager console and Create a Secret Key by following these steps - https://docs.aws.amazon.com/secretsmanager/latest/userguide/create_secret.html . That's it. You will soon receive a notification about the key you created. Now any event (Like Update or Delete) happens related to that Secret Key, you will receive the notification.",
```

Comment:
i think the comma at the end of this line is the one causing problems, remove it and we can run the validation again.

Reply:
I have removed it and validation successfully passed